CMakeLists.txt

```diff
@@ -5,11 +5,16 @@ project(projectname)
 list(APPEND CMAKE_MODULE_PATH "${PROJECT_SOURCE_DIR}/lib/cmake-modules")
 
 if(UNIX)
-	set(CMAKE_CXX_FLAGS "-m32 -fvisibility=hidden")
+	set(CMAKE_CXX_FLAGS "-m32 -fvisibility=hidden -std=c++11")
```

Comment:
Since this is a boilerplate, I don't think this constraint is necessary - users can decide if they need C++ 11 for their own plugin.

Reply:
Do we not think we should atleast provide a default? I mean they can change it but without it attempting to build the current project fails.